altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
Sure, ideally, we would wait until you started to prototype that stuff on
the vega-lite side and build it in parallel.

On Sun, Oct 18, 2015 at 1:28 AM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In altair/api.py
> <https://github.com/ellisonbg/altair/pull/68#discussion_r42317220>:
>
> > @@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
> >              self.shape._infer_type(self.parent.data)
> >
> >
> > +class Filter():
>
> It would be good to know when do you plan to test your composition API?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/68/files#r42317220>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com